### ISSUE TITLE
fenced code blocks are incorrectly treated as inline code

### ISSUE THREAD

**Issue by gilmoskowitz** [tldr-pages/tldr-lint#105]
Multiline fenced code blocks in input incorrectly report TLDR103. The attached example file returns the following result:
```
Error: Parse error on line 10:
... [f]iles:```bash`tar cf {{{target.tar
--------------------^
Expecting 'NEWLINE', 'TEXT', 'DASH', 'BACKTICK', got 'COMMAND_TEXT'
specs/pages/passing/fenced-code-block.md:8: TLDR103 Command example is missing its closing backtick
```

Note that the language spec (in this case `bash`) is followed by a backtick in the output but not in the original input. The [parser](https://github.com/tldr-pages/tldr-lint/blob/main/tldr.yy#L66) and [lexer](https://github.com/tldr-pages/tldr-lint/blob/main/tldr.l#L211-L229) (and other places in the lexer) seem to assume that all examples will fit on one line. While that's an understandable goal for tldr, it just doesn't fit all situations. For example tldr-pages/tldr#6868 includes examples of program _input_, not just program execution.

Expected behavior: Multiline fenced code blocks should be handled as such. The three backticks should be treated as opening and closing markers, not as adjacent, incomplete inline code blocks.

[fenced-code-block.md](https://github.com/tldr-pages/tldr-lint/files/7305958/fenced-code-block.md)

**Comment by MasterOdin** [comment]
For organizational purposes, as a number of tldr maintainers don't follow this repo actively, we should have the debate on whether or not multiline examples make sense or not in the tldr PR itself, as the resulting conclusion would possibly affect updating the [Style Guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md) in some way.

**Comment by marchersimon** [comment]
So, even if we did allow multiline examples, and even if we did come up with regulations on when to use them and how to use them, it would still break most of the clients, because they expect the examples to be one line. We could try to update the clients of course, but most clients in the official distro repos aren't really maintained anymore, so that's probably not possible.

**Comment by gilmoskowitz** [comment]
tldr-pages/tldr#6868 has been updated to remove the multiline examples so this issue is no longer holding up merging that pull request. Feel free to close this unless the maintainers decide otherwise.

**Comment by MasterOdin** [comment]
Closing this until we have a set example of needing to support code blocks.

Thanks for raising the issue (even if it is getting closed) and creating those pages, and gamely participating in the discussion on the styling with maintainers.
